engine/src/main/java/org/destinationsol/assets/sound/SpecialSounds.java

```diff
@@ -106,4 +107,48 @@ public void playColl(SolGame game, float absImpulse, SolObject o, Vector2 positi
         }
         game.getSoundManager().play(game, metal ? metalColl : rockColl, position, o, absImpulse * Const.IMPULSE_TO_COLL_VOL);
     }
+
+    /**
+     * Gets the damage sound associated with the given {@link MaterialType} and {@link DmgType}. If no sound is defined,
+     * null is returned.
+     *
+     * @param materialType the material type of the damaged entity
+     * @param damageType   the type of damage done
+     * @return the sound of the damage
+     */
+    public PlayableSound getHitSound(MaterialType materialType, DmgType damageType) {
```

Comment:
Why does this deserve to exist if `hitSound(metal, dmgType)` is already a thing?